docs/internal.md

```diff
@@ -42,17 +48,26 @@ It is a two-step process to establish persistent connections between clients and
 
 1. Client sends a negotiate request to application server.
 With Azure SignalR Service SDK, application server will return a redirect negotiation response with SignalR Service's URL and access token.
-Typical redirect response looks like as following:
-
-    ```json
-    {
-        "url":"https://test.service.signalr.net:5001/client/?hub=chat&...",
-        "accessToken":"<a typical JWT token>",
-        "availableTransports":[]
-    }
-    ```
-
+    * For ASP.NET Core SignalR, a typical redirect response looks like as following:
+
+        ```json
+        {
+            "url":"https://test.service.signalr.net/client/?hub=chat&...",
+            "accessToken":"<a typical JWT token>",
+            "availableTransports":[]

```

Comment:
"availableTransports" isn't supposed to be part of the redirect response
https://github.com/aspnet/SignalR/blob/release/2.2/specs/TransportProtocols.md#post-endpoint-basenegotiate-request